src/Core/Silk.NET.BuildTools/Bind/ClassWriter.cs

```diff
@@ -315,7 +315,7 @@ public static void WriteMixedModeClasses(this Project project, Profile profile,
                         sw.Write($"    public unsafe partial class {name}");
                         if (i.Functions.Any())
                         {
-                            sw.WriteLine($" : NativeExtension<{@class.ClassName}>");
+                            sw.WriteLine($" : NativeExtension<{project.NativeApiSetName}>");
```

Comment:
I don't think this is the best solution. Given class names should always be equal to the core class (the clang backend is bugged here, but this is a precedent set by the Khronos constructors) we should be able to use ClassName no problem by just fixing clang to use the core name for its classes always.

Reply:
i just did what you had mentioned on discord lol, how would we ensure that the profiles are parsed in the correct order? as its not too unreasonable for us to mess up later on and accidentally put the core profile after something else (unless that should be a detected case where we error out?)
If we can ensure the order, than it wouldnt be too hard, so i think ill do that